src/QMCWaveFunctions/Fermion/MultiSlaterDeterminantWithBackflow.cpp

```diff
@@ -834,12 +836,14 @@ void MultiSlaterDeterminantWithBackflow::evaluateDerivatives(ParticleSet& P,
           int upC        = C2node_up[ip];
           int dnC        = C2node_dn[ip];
           ValueType cdet = detValues_up[upC] * detValues_dn[dnC] * psiinv;
-          convert(cdet, dlogpsi[kk]);
+          //convert(cdet, dlogpsi[kk]);
+          dlogpsi[kk] = cdet;
```

Comment:
Please delete this commented line and all similar cases within this file.

Reply:
deleted.